great_expectations_cloud/agent/agent.py

```diff
@@ -123,9 +128,9 @@ def _listen(self) -> None:
             if subscriber is not None:
                 subscriber.close()
 
-    def _get_current_gx_agent_version(self) -> str:
-        version: str = metadata_version(self._PYPI_GX_AGENT_PACKAGE_NAME)
-        print(f"GX Agent version: {version}")
+    @classmethod
+    def _get_current_gx_agent_version(cls) -> str:
```

Comment:
I have to call it with `self` because `__init__` is an instance method.
I can call a class method from an instance method but I can't call an instance method from a class method.